server.js

```diff
@@ -49,15 +49,27 @@ app.post('/api/register', async (req, res) => {
   }
 });
 
-app.listen(port, () => {
-  console.log(`Server is running on port ${port}`);
-});
-
 // app.post('/api/achievement', async (req, res) => {
 //   console.log("It's alive!");
 //   res.status(200).json({ message: 'Achievement successful' });
 // });
 
+app.post('/api/attempt', async (req, res) => {
+  try {
+    const { characters_attempted, characters_missed, wpm, email } = req.body;
+    // console.log({ characters_attempted, characters_missed, wpm, email })
+    const getUserId = await getUserIdByEmail(email);
+    const result = insertAttempt(getUserId, characters_attempted, characters_missed, wpm);
+    res.status(200).json(result);
+  } catch (err) {
+    console.error("Attempt insertion error:", err);
```

Comment:
Nit: There is a pattern that I really like for log messages, which is to put variables in brackets.

This way, if `err` is ever an empty string, it is clear as you will just see `[]`. Also, it is just more clear.

```suggestion
    console.error(`Failed to insert record with error [${err}]`);
```

Reply:
Thanks!

Reply:
Totally. If you agree with this, you could potentially adopt this pattern for all lock messages. I'll leave that up to you.